server/server.js

```diff
@@ -26,7 +26,10 @@ const config = {
 // and /callback routes to the baseURL
 app.use(auth(config));
 
+// serve client to server
 app.use(express.static(path.join(__dirname, '../dist')));
+// routers
+app.use('/api', routes);
```

Comment:
i don't think we currently have a routes variable in this file so we'll have an error but other than that, this is good